services/frontend-v3/src/components/profileCards/ProfileCardsView.jsx

```diff
@@ -92,12 +93,14 @@ function ProfileCardsView({
                 <Switch
                   checkedChildren={<EyeOutlined />}
                   unCheckedChildren={<EyeInvisibleOutlined />}
-                  checked={disabled}
+                  checked={disabled && !forceDisabled}
```

Comment:
why do we need this?

Reply:
I added a message for the exfeeder card to state that the user is not an ex-feeder so that they can be able to edit this value. I assume that we would still want to never show this information publicly so I made a way to force a card to stay hidden and I do this is they have ex-feeder unchecked